files/scripts/asic_status.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+function check_asic_status() {
+    python3 /usr/local/bin/asic_status.py $DEV
```

Comment:
good to remove python3 hardcoding here ? IT will anyways use that with the #!/usr/bin/env python3 in asic_status.py file.

Reply:
Removed.